tensorflow/lite/kernels/register.cc

```diff
@@ -356,6 +357,9 @@ BuiltinOpResolver::BuiltinOpResolver() {
   AddBuiltin(BuiltinOperator_NOT_EQUAL, Register_NOT_EQUAL(),
              /* min_version */ 1,
              /* max_version */ 2);
+  AddBuiltin(BuiltinOperator_SOFTPLUS, Register_SOFTPLUS(),
+             /* min_version */ 1,
```

Comment:
These aren't necessary. Softplus is a new op so it has only one version

Reply:
@miaout17 , thanks for the review i have updated the comments as per your suggestion.

Regards
Amit